virtio-devices/src/net.rs

```diff
@@ -545,6 +545,7 @@ impl Net {
         id: String,
         fds: &[RawFd],
```

Comment:
nit: typo of default in commit message. I also think your commit message doesn't say what the code does or why

```
net: Don't override default TAP interface MTU

Adjust MTU logic such that:
1. Apply an MTU to the TAP interface if the user supplies it
2. Always query the TAP interface for the MTU and expose that.
```

Reply:
Ok!